blocks/agent-transactions/agent-transactions.js

```diff
@@ -0,0 +1,105 @@
+import {
+  table, tbody, th, thead, tr, td, h1, a,
+} from '../../scripts/dom-helpers.js';
+import { getMetadata } from '../../scripts/aem.js';
+
+const getClosedTransactions = async () => {
+  const agentId = getMetadata('agent-id');
+  const formattedData = [];
+
+  try {
+    const response = await fetch(`https://www.commonmoves.com/bin/bhhs/agentPropertyListingsServlet.${agentId}.json`);
```

Comment:
I've merged the proxy logic into main. Please update this to use a relative path, and test by setting up local proxy as per the README

Reply:
Verified and its working fine